theworkoutinfo/Repositories/ContentRepository.swift

```diff
@@ -37,5 +37,31 @@ class ContentRepository {
             return Observable.from(optional: result)
         })
     }
-
+    
+    func getExercisesByCategoryId(categoryId: Int, pageNumber: Int) -> Observable<ResultCollection<Exercise>> {
+        return RequestHelper.sharedInstance.performRequestCollection(service: .exercisesByCategory(categoryId: categoryId, pageNumber: pageNumber)).flatMap({ (result: ResultCollection<Exercise>) -> Observable<ResultCollection<Exercise>> in
```

Comment:
Is this not reusing the get exercises w e had before? If not we should try to do it.